src/m365/tenant/commands/people/people-profilecardproperty-remove.spec.ts

```diff
@@ -120,7 +120,20 @@ describe(commands.PEOPLE_PROFILECARDPROPERTY_REMOVE, () => {
       throw `Invalid request ${opts.url}`;
     });
 
-    await assert.doesNotReject(command.action(logger, { options: { name: 'StateOrProvince', force: true } }));
+    await command.action(logger, { options: { name: 'StateOrProvince', force: true } });
```

Comment:
I do think it's bad practice to have a test without an assert statement. even if the behavior is the same for these test cases, I'd suggest we keep them in.

Reply:
...if you dislike an assert that does not give new information, of course we can add an assert for something else. (For example that nothing is logged)

Reply:
True, haven't checked all tests to be honest. In this case I think we should assert that the delete request was actually fired. Right now we can totally remove the delete request from the code and the rest will still succeed.

Reply:
Fixed it.